maestro-android/src/androidTest/java/dev/mobile/maestro/MaestroDriverService.kt

```diff
@@ -396,7 +414,14 @@ class Service(
         uiDevice.pressKeyCode(keyCode, META_SHIFT_LEFT_ON)
     }
 
-    companion object {
-        private const val LENGTH_KEY_VALUE_PAIR = 2
+    internal fun Throwable.internalError() = Status.INTERNAL.withDescription(message).asException()
+
+    enum class FileType(val ext: String, val mimeType: String) {
+        JPG("jpg", "image/jpg"),
+        JPEG("jpeg", "image/jpeg"),
+        PNG("png", "image/png"),
+        GIF("gif", "image/gif"),
+        MP4("mp4", "video/mp4"),
+        MP3("mp3", "audio/mp3")
```

Comment:
Going to remove audio for this first release since it needs a little investigation for iOS

Reply:
Done ✅